db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,14 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :playbook_name, :string
```

Comment:
Do we have to add all these fields as most of them are inherited from job template?

For both orchestration stack and tower job, at runtime the provider accepts options and parameters. We already collect parameters and save in orchestration_stack_parameters, but so far we do not collect options. So we can either create orchestration_stack_options table; or add options field to the stack table. The simple solution is adding the options column, which is a serializable hash where you can store most of the fields you are adding here.